cmd2/utils.py

```diff
@@ -210,13 +213,16 @@ def is_text_file(file_path: str) -> bool:
     return valid_text_file
 
 
-def remove_duplicates(list_to_prune: List) -> List:
+_T = TypeVar('_T')
+
+
+def remove_duplicates(list_to_prune: List[_T]) -> List[_T]:
     """Removes duplicates from a list while preserving order of the items.
 
     :param list_to_prune: the list being pruned of duplicates
     :return: The pruned list
     """
-    temp_dict = collections.OrderedDict()
+    temp_dict: collections.OrderedDict[_T, Any] = collections.OrderedDict()
```

Comment:
We can remove OrderedDict usage now that we are on Python 3.6.

Reply:
Nevermind, we should keep OrderedDict for non-CPython environments where dictionaries may not preserve insertion order.